pkg/contracts/src/OsmoticController.sol

```diff
@@ -107,146 +79,63 @@ contract OsmoticController is Initializable, OwnableUpgradeable, PausableUpgrade
     function _authorizeUpgrade(address newImplementation) internal override onlyOwner {}
 
     /* *************************************************************************************************************************************/
-    /* ** Pool Creation Function                                                                                                       ***/
+    /* ** Creation Functions                                                                                                             ***/
     /* *************************************************************************************************************************************/
 
-    function createOsmoticPool(bytes calldata _poolInitPayload) external whenNotPaused returns (address pool_) {
```

Comment:
What's the rationale behind using `memory` instead of `calldata` here?

Reply:
I was having a compilation error because I was using the method internally and was not `calldata` in same cases.